pending/change-session-id.pod

```diff
@@ -0,0 +1,106 @@
+=head1 change_session_id
+
+It is considered good security practice, and is a requirement of a number
+of established security standards, to change session ID on any change of
+privilege level. For starters you should consider doing this on user login,
+and possibly at any other time a user has some privilege added or removed.
+
+=head2 Session destruction
+
+Destroying a session in L<Dancer2> has always been possible using
+L<Dancer2::Core::App/destroy_session> like so:
+
+    app->destroy_session;
+
+Session destruction by its very nature causes the session ID to change since
+the server instructs the client to forget the session ID, send it a new
+session ID and also destroys any session data held on the server.  This is
+considered a "good thing" when a user logs out of an application but it is
+not very useful when a user logs in and you want to keep existing session
+data such as cart products.
+
+=head2 change_session_id
+
+L<Dancer2::Core::App/change_session_id> to the rescue!
+
+If you are using a recent version of L<Dancer2::Plugin::Auth::Extensible>
```

Comment:
Agree that it is much better to include the version. Was off-net when I originally wrote this and meant to add in the version before I pushed but forgot. Will reword to say "If you are using version 0.610 or later of..." as that still reads reasonably well and I'm not a big fan of parentheses in prose.